plugins/hikvision/src/main.ts

```diff
@@ -26,13 +26,20 @@ class HikVisionCamera extends RtspSmartCamera implements Camera {
                 events.on('close', () => ret.emit('error', new Error('close')));
                 events.on('error', e => ret.emit('error', e));
                 events.on('event', (event: HikVisionCameraEvent, cameraNumber: string) => {
-                    // if (this.getRtspChannel() && cameraNumber !== this.getCameraNumber()) {
-                    //     return;
-                    // }
                     if (event === HikVisionCameraEvent.MotionDetected
                         || event === HikVisionCameraEvent.LineDetection
                         || event === HikVisionCameraEvent.FieldDetection) {
-                        this.motionDetected = true;
+                        if (this.getRtspChannel() && cameraNumber !== this.getCameraNumber()) {
+                            // this.console.error(`### Skipping motion event ${cameraNumber} != ${this.getCameraNumber()}`);
+                            return;
+                        }
+        
+                        // this.console.error('### Detected motion, camera: ', cameraNumber);
+                        const prevMotion = this.motionDetected;
+                        if (!prevMotion) { 
+                            this.motionDetected = true;
+                            this.console.log(`motionDetected, camera ${this.getCameraNumber()}`);
```

Comment:
This is unnecessary, as all state changes are automatically logged in the Events tab: 

<img width="457" alt="image" src="https://user-images.githubusercontent.com/73924/147865959-f363959b-4821-443b-85be-8ab449c8ab74.png">